modules/topicsFpdModule.js

```diff
@@ -232,17 +238,20 @@ export function hasGDPRConsent() {
 /**
  * function to load the iframes of the bidder to load the topics data
  */
-function loadTopicsForBidders() {
+export function loadTopicsForBidders() {
   HAS_GDPR_CONSENT = hasGDPRConsent();
+
   if (!HAS_GDPR_CONSENT || !HAS_DEVICE_ACCESS) {
     logInfo('Topics Module : Consent string is required to fetch the topics from third party domains.');
     return;
   }
+
   const topics = config.getConfig('userSync.topics') || bidderIframeList;
+
   if (topics) {
     listenMessagesFromTopicIframe();
     const randomBidders = getRandomBidders(topics.bidders || [], topics.maxTopicCaller || 1)
```

Comment:
@remysaissy i agree with you that we shouldn't default to 1 (I think that was added in there during an initial PR for this, as the logic around Topics is still being figured out..) Thinking that maybe the default for this module should be off?  aka no publishers?  since this module requires config setup anyway.  Or, maybe if there is some sort of prebid general iframe/fetch url to point to as a default?  not sure how that would be exactly

Reply:
Following @patmmccann's remark on feature request discussion to be in issues, I've followed-up on this on the issue: https://github.com/prebid/Prebid.js/issues/9981#issuecomment-1649241119